### ISSUE TITLE
About domain transfer settins

### ISSUE THREAD

**Issue by LiewFeng** [naoto0804/cross-domain-detection#28]
Hi, @naoto0804  I wonder the specific settings of domain transfer method.
For example,
train A: images of  VOC2007, the trainval part
train B: images of clipart, the train part
test A uses the same images as train A, test B uses the same images as train B
Is it right?

**Comment by naoto0804** [comment]
Test B uses the test part of clipart images.

**Comment by LiewFeng** [comment]
@naoto0804 Sorry for my delayed reply. I checked the code of CycleGAN. It seems that the setting of Test B doesn't affect the results? Because the model has already been trained. 
And I couldn't reproduce the results. The learning rate is 2e-4 as  [this](https://github.com/naoto0804/cross-domain-detection/issues/16) suggests.
I used [pytorch version CycleGAN](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix) and default parameters.

**Comment by LiewFeng** [comment]
10 epochs for lr=2e-4,  and 10 epochs for decay

**Comment by naoto0804** [comment]
I think 'the setting of Test B' shouldn't affect the results. In the domain adaptation setting, as usual, we only have access to the train images in the target domain during training.

**Comment by naoto0804** [comment]
It's true that the CycleGAN is nice but sometimes unstable;

**Comment by LiewFeng** [comment]
@naoto0804 I tried several times, but still got inferior results. Are there any settings of mine wrong? Could you please check parameters one more time? Lots of thanks.
----------------- Options ---------------
               batch_size: 1                             
                    beta1: 0.5                           
          checkpoints_dir: ./checkpoints                 
           continue_train: False                         
                crop_size: 256                           
                 dataroot: ./datasets/dt_clipart         	[default: None]
             dataset_mode: unaligned                     
                direction: AtoB                          
              display_env: main                          
             display_freq: 400                           
               display_id: -1                            	[default: 1]
            display_ncols: 4                             
             display_port: 8097                          
           display_server: http://localhost              
          display_winsize: 256                           
                    epoch: latest                        
              epoch_count: 1                             
                 gan_mode: lsgan                         
                  gpu_ids: 0,1                           	[default: 0]
                init_gain: 0.02                          
                init_type: normal                        
                 input_nc: 3                             
                  isTrain: True                          	[default: None]
                 lambda_A: 10.0                          
                 lambda_B: 10.0                          
          lambda_identity: 0.5                           
                load_iter: 0                             	[default: 0]
                load_size: 286                           
                       lr: 0.0002                        
           lr_decay_iters: 50                            
                lr_policy: linear                        
         max_dataset_size: inf                           
                    model: cycle_gan                     
                 n_epochs: 10                            	[default: 100]
           n_epochs_decay: 10                            	[default: 100]
               n_layers_D: 3                             
                     name: clipart_cyclegan              	[default: experiment_name]
                      ndf: 64                            
                     netD: basic                         
                     netG: resnet_9blocks                
                      ngf: 64                            
               no_dropout: True                          
                  no_flip: False                         
                  no_html: False                         
                     norm: instance                      
              num_threads: 8                             	[default: 4]
                output_nc: 3                             
                    phase: train                         
                pool_size: 50                            
               preprocess: resize_and_crop               
               print_freq: 100                           
             save_by_iter: False                         
          save_epoch_freq: 5                             
         save_latest_freq: 5000                          
           serial_batches: False                         
                   suffix:                               
         update_html_freq: 1000                          
                  verbose: False                         
----------------- End -------------------
One of your result and my result

![000012](https://user-images.githubusercontent.com/42316773/80697743-fcc6a780-8b0b-11ea-878b-e500a383b677.jpg)
![000012_fake_B](https://user-images.githubusercontent.com/42316773/80697761-05b77900-8b0c-11ea-9b7d-8095cbcb6142.png)

**Comment by Fly-dream12** [comment]
Why is the quantity of domain transfered data is 16651, which is about 33 times of the original clipart dataset. And how is the annotation of transferred dataset  generated, should it be the same annotation of the source clipart images? @naoto0804 
Have you solved this issue? @PhoneSix
